glean_parser/parser.py

```diff
@@ -58,6 +58,10 @@ def _load_metrics_file(filepath):
         yield f"{filepath}: {str(e)}"
         return {}
 
+    if metrics_content is None:
+        yield f"{filepath}: No JSON content found."
```

Comment:
Nit: YAML content vs. JSON content.

I'd change the message to be more about what we want from the user (instead of the technical requirement).
Maybe something like:
> Metrics file can't be empty. Please supply a valid YAML file for: {filepath}

Reply:
Sure thing.  Done.